Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingList/ManagePaywallButton.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ManagePaywallButton.swift
+//  PaywallsTester
+//
+//  Created by James Borthwick on 2024-04-30.
+//
+
+import SwiftUI
+
+struct ManagePaywallButton: View {
+
+    enum Kind: String {
+        case edit
+        case new
+
+        var systemImageName: String {
+            switch self {
+            case .edit:
+                return "slider.horizontal.2.square.on.square"
+            case .new:
+                return "plus.square.on.square"
+            }
+        }
```

Comment:
if we're going in this direction, might as well lean in and also add a `var buttonName: String` so you don't need the `buttonName ?? "Edit Paywall"` line later on

Reply:
Good idea, I added a new default name here. We need to keep the `??` though, because the constructor takes an optional button name, which we use for Offerings without paywalls, where the button name is the name of the Offering.